public/main/services/sparkmax-service.ts

```diff
@@ -120,17 +122,22 @@ onTwoWayCall("start-server", (cb, port: any) => {
   }
   const relPath = dllFolder + "sparkmax" + (isWin ? ".exe" : "");
   const exePath = path.join(__dirname, relPath);
+  const args = ["-r", "-p", port, "--verbosity", VERBOSITY, "--log", LOG];
+  const cmd = `${exePath} ${args.join(" ")}`;
+  const onError = once((e) => {
+    console.log(`There was an error starting the SPARK server: ${cmd}` + e);
+    cb(e);
+  });
   if (fs.existsSync(exePath)) {
     try {
-      usbProc = execFile(exePath, ["-r", "-p", port], (error: any, stdout: any) => {
-        console.log("Successfully started the SPARK server. PID is " + (usbProc as ChildProcess).pid);
-        cb(error);
-      });
-      console.log("Successfully started the SPARK server.");
+      console.log(`Run SPARK server executable: ${cmd}`);
+      usbProc = spawn(exePath, args);
+      usbProc.stdout.pipe(createWriteStream("server-out.log"));
```

Comment:
@aleksey-mitar 

Should this location default to the same as the application logs instead? Otherwise this will install to Program Files/ and not be writable